packages/runtime/src/useCases/transport/relationshipTemplates/CreateOwnRelationshipTemplate.ts

```diff
@@ -78,6 +78,10 @@ export class CreateOwnRelationshipTemplateUseCase extends UseCase<CreateOwnRelat
 
         if (!(transformedContent instanceof RelationshipTemplateContent)) return;
 
+        if (transformedContent.onNewRelationship.expiresAt?.isAfter(templateExpiresAt)) {
```

Comment:
We could also validate that the expiration date is in the future

Reply:
Also if there is an expiration date for the template, there must be one for the request as well